components/PageNavigation.vue

```diff
@@ -295,6 +296,16 @@
             color: $white;
           }
         }
+        span {
+          overflow: hidden;
+          white-space: nowrap;
+          display: block;
```

Comment:
Remove this display: block. It overwrites the above (line 222) set display: flex which keeps the icon and text properly aligned and the element height correct.